trs_filer/ga4gh/trs/endpoints/registerObjects.py

```diff
@@ -0,0 +1,73 @@
+"""Controller for adding new objects."""
+
+from random import choice
+from typing import Dict
+from trs_filer.app import logger
```

Comment:
Not so good. It is convention to create a logger for each module with:

```
import logging
# your other imports

logger = logging.getLogger(__name__)
```

Do _not_ import loggers from other modules.

Please check the [logging documentation](https://docs.python.org/3/library/logging.html) for more details why that is recommended.